dynamo-lib/src/cmr/dynamo/config.clj

```diff
@@ -0,0 +1,11 @@
+(ns cmr.dynamo.config
+  "Contains functions for retrieving important DynamoDB details from environment variables"
+  (:require [cmr.common.config :as cfg :refer [defconfig]]))
+
+(defconfig dynamo-table
+   "DynamoDB table for meta-metadata items"
+   {:default "meta-metadata"})
+
+(defconfig dynamo-toggle
+  "Three-way toggle for DynamoDB functionality. 'dynamo-off' uses only Oracle, 'dynamo-on' uses both Oracle and EFS, and 'dynamo-only' uses only EFS"
```

Comment:
Is this comment correct?  dynamo-off means that only oracle is used and not EFS and/or Oracle?
dynamo-on - shouldn't dynamo be used and not oracle? or are all three being used?
dynamo-only uses only EFS?  not Dynamo?

Reply:
Oh, yeah that's wrong. I'd copied some stuff from elsewhere and hadn't realized that part hadn't been updated.